domain/machine/service/service.go

```diff
@@ -61,9 +61,9 @@ type State interface {
 	// for the machine cloud instances.
 	InitialWatchInstanceStatement() (string, string)
 
-	// InstanceId returns the cloud specific instance id for this machine.
+	// InstanceID returns the cloud specific instance id for this machine.
 	// If the machine is not provisioned, it returns a NotProvisionedError.
```

Comment:
**todo:** Qualify the godoc link [machineerrors.NotProvisionedError]

Reply:
Fixed in 9b4a2b06aa357bd3119ef601318c0f923c583ff7.